kubernetes/services/conductor.yml.j2

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind:       Deployment
+metadata:
+  name: conductor
+spec:
+  replicas:             2
+  revisionHistoryLimit: 2
+  template:
+    metadata:
+      labels:
+        run: conductor
+    spec:
+      containers:
+        - name:             conductor
+          image:            {{ docker_registry }}/{{image_prefix }}concent-api:{{ image_version }}
+          imagePullPolicy:  Always
+          ports:
+            - containerPort: 80
+          resources:
+            requests:
+              memory: "0.5Gi"
+              cpu:    "50m"
+            limits:
```

Comment:
Let's try to give Django a bit less memory. Try 100 MB per gunicorn worker (so 200 MB for each pod). This application is not big enough to eat that much memory per pod.

Reply:
Done.

Reply:
Have you tested it? Does it work?